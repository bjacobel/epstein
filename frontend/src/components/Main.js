import React, { Component } from 'react';

import styles from '../stylesheets/main.css';

export default class Main extends Component {
  render() {
    return <></>;
  }
}
