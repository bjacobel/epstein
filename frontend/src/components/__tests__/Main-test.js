import React from 'react';
import { shallow } from 'enzyme';

import Main from 'components/Main';

describe('main component', () => {
  describe('un-Connected component', () => {
    it('matches snapshot', () => {
      expect(shallow(<Main />)).toMatchSnapshot();
    });
  });
});
