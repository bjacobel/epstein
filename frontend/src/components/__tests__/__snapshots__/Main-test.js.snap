// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main component un-Connected component matches snapshot 1`] = `<Fragment />`;
